specification/1.0/README.md

```diff
@@ -105,12 +105,10 @@ Example (minimal sdTF):
 
 ## Nodes
 
-Trees in sdTF are made of nodes. Nodes can reference other nodes and/or data items. They can have an optional name, optional attributes, and an optional type hint. 
+Trees in sdTF are made of nodes. Nodes can reference other `nodes` and/or data `items`. They can have an optional `name`, optional `attributes`, and an optional `typeHint`. 
 Due to the flat hierarchy it's theoretically possible to create circular structures of nodes, but that's discouraged. 
 
-Examples: 
-
-### Node referencing data items
+### Example: Node referencing data items
 
```

Comment:
I would recommend adding some description here with a rough explanation of the individual attribute values.
Like, "A root level node element of the type specified at [typeHint[2]](#reference-typehint), consisting of 13 elements".

Reply:
@matthiasreisacher very good idea! Could you do it please? I suggest to not call it "root level node", because we use chunks as the root nodes.